spec/forms/hub/outbound_call_form_spec.rb

```diff
@@ -72,12 +72,11 @@
   end
 
   context "twiml" do
```

Comment:
Move tests for this into the testing for #dial, expecting the twilio_client object to receive this parameter.